### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by sjpadgett**
- remove cards max height to prevent scroll on longer lists.

Fixes #

### CODE REVIEW

**Comment by bradymiller** [comment]
dance-bear

File: interface/patient_file/summary/demographics.php [hunk 1]
```diff
@@ -835,7 +835,21 @@ function openReminderPopup() {
...
+            // test ensure at least an element we want.
+            if (target.classList.contains("collapse")) {
+                // who is icon. Easier to catch BS event than create one specific for this decision..
+                // Should always be icon target
+                let iconTarget = e.target.children[0] || e.target;
+                // toggle
+                if (iconTarget.classList.contains("fa-expand")) {
+                    iconTarget.classList.remove('fa-expand');
+                    iconTarget.classList.add('fa-compress');
+                }
```
Comment (stephenwaite) [inline interface/patient_file/summary/demographics.php:847]: Jswizard

**Comment by sjpadgett** [comment]
@bradymiller Merging this so I can test on demo tomorrow. Ill do PR for the demographics_ps_expand persist tomorrow!

**Comment by bradymiller** [comment]
tenor99

**Comment by bradymiller** [comment]
Going to bring this into rel-702 to expedite

**Pull request merged**